src/modules/integrator-guide/pages/authentication-security/auth.adoc

```diff
@@ -105,6 +105,7 @@ The scope is defined as a series of role memberships that are either *permitted*
 |===
 | [#auth-scope-permitted]#<<auth-scope-permitted,Permitted roles>>#
```

Comment:
Uhh, I messed up the table definition and that's why everything is bold and stuff is missing...
Can you add a `| ` before the `Permitted roles allow you to ...`, `Prohibited roles allow you to ...`, and `Forced roles allow you to  ...`; so `| Permitted roles allow you to ...` for example.